packages/cli/src/commands/screenshot-diff/screenshot-diff.command.ts

```diff
@@ -111,6 +233,16 @@ export const diffScreenshots: (options: {
       value: 1,
     });
   }
+
+  return comparisonResults;
+};
+
+const logComparisonResultMessage: (
+  logger: Logger,
+  message: string,
+  outcome: ComparisonOutcome
+) => void = (logger, message, outcome) => {
+  logger.info(`${message} => ${outcome === "fail" ? "FAIL!" : "PASS"}`);
```

Comment:
nitpick: more foolproof if there are more outcome types later.

```suggestion
  logger.info(`${message} => ${outcome === "pass" ? "PASS" : "FAIL!"}`);
```